icaruscode/CRT/CRTPMTMatchingAna_module.cc

```diff
@@ -123,31 +211,81 @@ class icarus::crt::CRTPMTMatchingAna : public art::EDAnalyzer {
   TTree* fMatchTree;
 
   //matchTree vars
-  int                    fNCrt;         //number of CRT hits per event
-  vector<vector<double>> fCrtXYZT;      //CRT hit x,y,z,t [cm/ns]
-  vector<vector<double>> fCrtXYZErr;    //CRT hit x,y,z,t error [cm/ns]
-  vector<int>            fCrtRegion;    //CRT hit region code
-  vector<double>         fCrtPE;        //total PE's for CRT hit
+
+  vector<double>      fOpFlashPos; // Position of the optical Flash Barycenter
+  double         fOpFlashTime; // Time of the optical Flash w.r.t. Global Trigger
+  double         fOpFlashTimehit;
+  double	fOpFlashTimeAbs;
+  bool         fInTime; // Was the OpFlash in beam spill time?
+  bool		fInTime_gate;
+  bool 		fInTime_beam;
+  double         fOpFlashPE; // Total PEs of the optical Flash
+  vector<double>      fOpHitX; // X position of the OP hit
+  vector<double>      fOpHitY; // Y position of the OP hit
+  vector<double>      fOpHitZ; // Z position of the OP hit
+  vector<double>      fOpHitT; // T of the OP hit
+  vector<double>      fOpHitA; //Amplitude (ADC) of the OP hit
+  int              fNCRTmatch;
+  vector<vector<double>>       fMatchedCRTpos; // Position of the matched CRTs
+  //vector<double[3]>       fMatchedCRTpos_err; // Position error of the matched CRTs
+  vector<double>          fMatchedCRTtime; // Time of the matched CRT Hits w.r.t. Global Trigger
```

Comment:
If fMatchedCRTtime uses µs rather than ns, I suggest adding the units to the variable name, i.e. fMatchedCRTtime_us

Reply:
changed the name!